x-pack/plugins/metrics_entities/server/modules/index.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { hostMetrics, hostMetricsMapping } from './host_metrics';
+import { userMetrics, userMetricsMapping } from './user_metrics';
+import { ipMetrics, ipMetricsMapping } from './network_metrics';
+import { hostEntities, hostEntitiesMapping } from './host_entities';
+import {
+  destinationCountryIsoCodeEntities,
+  destinationCountryIsoCodeEntitiesMapping,
+  destinationIpEntities,
+  destinationIpEntitiesMapping,
+  sourceCountryIsoCodeEntities,
+  sourceCountryIsoCodeEntitiesMapping,
+  sourceIpEntities,
+  sourceIpEntitiesMapping,
+} from './network_entities';
+import { Mappings, Transforms } from './types';
+import { userEntities, userEntitiesMapping } from './user_entities';
+
+/**
+ * These module names will map 1 to 1 to the REST interface.
+ */
+export enum ModuleNames {
+  hostSummaryMetrics = 'host_metrics',
+  hostSummaryEntities = 'host_entities',
+  networkSummaryEntities = 'network_entities',
+  networkSummaryMetrics = 'network_metrics',
+  userSummaryEntities = 'user_entities',
+  userSummaryMetrics = 'user_metrics',
+}
```

Comment:
Good stuff with the breakout here! 🙂  As discussed with @FrankHassanabad, these aren't solutions-orientated as the hope is these transforms can be used by all solutions across the stack, but with how this is laid out solutions could namespace their own transforms via modules as needed.

Reply:
Yep, see comment above about this: https://github.com/elastic/kibana/pull/96446#discussion_r617077128